services/openvpn/service/manager.go

```diff
@@ -94,68 +83,46 @@ func (manager *Manager) Start() (err error) {
 
 	currentCountry, err := manager.locationResolver.ResolveCountry(publicIP)
 	if err != nil {
-		return err
+		return
```

Comment:
how error should be handled here?

Reply:
Implemented custom error `ErrorLocation` and special warning recommendation for Provider